lib/galaxy/util/plugin_config.py

```diff
@@ -78,8 +69,5 @@ def plugin_source_from_path(path):
 
 
 def __read_yaml(path):
-    if yaml is None:
-        raise ImportError("Attempting to read YAML configuration file - but PyYAML dependency unavailable.")
-
```

Comment:
I guess the changes to this file should be reverted (apart from the first try-except removal) to be coherent with `lib/pulsar/client/config_util.py` .